pkgs/applications/networking/browsers/brave/default.nix

```diff
@@ -29,6 +30,7 @@
 , libXrender
 , libXScrnSaver
 , libXtst
+, mesa_drivers
```

Comment:
That should make OfBorg happy without affecting the actual derivation (assuming that's required) (1/2)
```suggestion
, mesa
```